OpenRA.Mods.Common/Util.cs

```diff
@@ -84,16 +84,17 @@ public static WAngle AngleDiffToStep(WAngle facing, int numFrames)
 			return new WAngle(a % step - step / 2);
 		}
 
-		public static WAngle GetInterpolatedFacing(WAngle facing, int facings, int interpolatedFacings)
+		/// <summary>Returns the angle that the closest facing sprite should be rotated by to achieve the closest interpolated facing.</summary>
+		public static WAngle GetInterpolatedFacingRotation(WAngle facing, int facings, int interpolatedFacings)
```

Comment:
I don't think the name should be changed to `Get...Rotation` unless it is in fact returning a `WRot`. You could call it `Get...Angle` if you wanted to specify the returned object type in the method name.

Reply:
Rotation is used everwhere else in the call chain (`Sequence.GetSpriteWithRotation`, `Animation.Render`, `SpriteRenderable`). IMO its clear from the context that this is a 2D rotation of a sprite in screen-space, however, if there is a consensus that the naming should be changed it should be done in its own PR (this one is already too big!).